### PR TITLE
Update the diagrams

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: doc/integrator/docker.rst [hunk 1]
```diff
@@ -8,14 +8,112 @@ Architecture schema
...
+        subgraph Project composition
+        haproxy[haproxy];
+        geoportal[GeoMapFish server<br>camptocamp/geomapfishapp-geoportal];
+        tcc[TileCloud-chain<br>camptocamp/tilecloud-chain];
+        tccs[TileCloud-chain slave<br>camptocamp/tilecloud-chain];
+        mapserver[MapServer<br>camptocamp/mapserver<br>or/and<br>camptocamp/qgis-server];
+        tinyows[tinyows];
+        print[Mapfist Print<br>camptocamp/mapfish-print];
+        alembic[GeoMapFish alembic<br>camptocamp/geomapfishapp-geoportal<br>minor databage upgrade];
+        config[Project config<br>camtocamp/%project%-config<br>injected the config in all the containers];
```
Comment (arnaud-morvan) [inline doc/integrator/docker.rst:77]:
```suggestion
        config[Project config<br>camtocamp/%project%-config<br>provides the config for all other containers];
```

File: doc/integrator/docker.rst [hunk 2]
```diff
@@ -8,14 +8,112 @@ Architecture schema
...
+        client(Web client);
+
+        k8s[Kubernetes routing];
+        geoportal[GeoMapFish server<br>camptocamp/geomapfishapp-geoportal];
+        tcc[TileCloud-chain<br>camptocamp/tilecloud-chain];
+        tccs[TileCloud-chain slave<br>camptocamp/tilecloud-chain];
+        mapserver[MapServer<br>camptocamp/mapserver<br>or/and<br>camptocamp/qgis-server];
+        tinyows[tinyows];
+        print[Mapfist Print<br>camptocamp/mapfish-print];
+        alembic[GeoMapFish alembic<br>camptocamp/geomapfishapp-geoportal<br>minor databage upgrade];
```
Comment (arnaud-morvan) [inline doc/integrator/docker.rst:24]:
```suggestion
        alembic[GeoMapFish alembic<br>camptocamp/geomapfishapp-geoportal<br>minor database upgrades];
```

File: doc/integrator/docker.rst [hunk 3]
```diff
@@ -8,14 +8,112 @@ Architecture schema
...
+        config[Project config<br>camtocamp/%project%-config<br>injected the config in all the containers];
```
Comment (arnaud-morvan) [inline doc/integrator/docker.rst:25]:
```suggestion
        config[Project config<br>camtocamp/%project%-config<br>provides the config for all other containers];
```

File: doc/integrator/docker.rst [hunk 4]
```diff
@@ -8,14 +8,112 @@ Architecture schema
...
+        apache[Apache];
+        subgraph Project composition
+        haproxy[haproxy];
+        geoportal[GeoMapFish server<br>camptocamp/geomapfishapp-geoportal];
+        tcc[TileCloud-chain<br>camptocamp/tilecloud-chain];
+        tccs[TileCloud-chain slave<br>camptocamp/tilecloud-chain];
+        mapserver[MapServer<br>camptocamp/mapserver<br>or/and<br>camptocamp/qgis-server];
+        tinyows[tinyows];
+        print[Mapfist Print<br>camptocamp/mapfish-print];
+        alembic[GeoMapFish alembic<br>camptocamp/geomapfishapp-geoportal<br>minor databage upgrade];
```
Comment (arnaud-morvan) [inline doc/integrator/docker.rst:76]:
```suggestion
        alembic[GeoMapFish alembic<br>camptocamp/geomapfishapp-geoportal<br>minor database upgrades];
```

**Review by arnaud-morvan — APPROVED** [review]

File: doc/integrator/docker.rst (hunk 1 above)
Comment (sbrunner) [reply doc/integrator/docker.rst:77]: fixed :-)

File: admin/config.yaml [hunk 5]
```diff
@@ -62,7 +62,7 @@ vars:
         relevant_for:
           - layergroup
       # Layer
-      - name: copyable
+      - name: copiable
```
[the remark this answers is not in the document]
Comment (sbrunner) [reply admin/config.yaml]: ? Effectively... Thanks :-)

**Pull request merged**